test/test_ops.py

```diff
@@ -727,20 +727,20 @@ def test_bbox_xywh_cxcywh(self):
         self.assertEqual(box_xywh.dtype, box_tensor.dtype)
         assert torch.all(torch.eq(box_xywh, box_tensor)).item()
 
-    # def test_bbox_convert_jit(self):
-    #     box_tensor = torch.tensor([[0, 0, 100, 100], [0, 0, 0, 0],
-    #                               [10, 15, 30, 35], [23, 35, 93, 95]], dtype=torch.float)
+    def test_bbox_convert_jit(self):
```

Comment:
I was wondering if it would be more simple to check scripted function in all your `test_bbox_*`:
```python
scripted_fn = torch.jit.script(ops.box_convert)

for fn in [ops.box_convert, scripted_fn]:
    # ...
```

Reply:
Possible, this PR is open for a lot of refactoring 👍 .